### PR TITLE
fix: bootstrapping procedure

### PR DESCRIPTION
**Pull request by linuxbandit**
another hackathon, another review :D

### CODE REVIEW

File: vagrant-post-script/generate_secrets.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 CURDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 ## Create secrets
-echo -e "[MyAEGEE] Creating random secrets if not existing"
+echo -e "[MyAEGEE] Creating *very* random secrets if not existing"
 
 ## Create a new random jwt key
 if ! [[ -f "${CURDIR}/../secrets/jwt_key" ]]; then
```
Comment (serge1peshcoff) [inline vagrant-post-script/generate_secrets.sh:9]: can you check which secrets are actually needed? I think jwt_key may be not used as well as others

File: vagrant-post-script/install_other_utils.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "[Vagrant] ###################     Installing nvm"
+curl -sS -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+
+
+echo "[Vagrant] ###################     Installing make because apparently it's not standard no more??"
```
Comment (WikiRik) [inline vagrant-post-script/install_other_utils.sh:7]: make is indeed not standard anymore in Ubuntu 20.04. Shall we change the image in Vagrantfile also to Ubuntu 20.04 then?

File: vagrant-post-script/install_other_utils.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "[Vagrant] ###################     Installing nvm"
+curl -sS -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
```
Comment (WikiRik) [inline vagrant-post-script/install_other_utils.sh:4]: https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh is the newest version

File: vagrant-post-script/install_docker_composer.sh [hunk 4]
```diff
@@ -9,6 +9,3 @@ if [ -a /usr/local/bin/docker-compose ]; then echo "[Vagrant] ##################
 curl -sSL "https://github.com/docker/compose/releases/download/1.24.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```
Comment (WikiRik) [inline vagrant-post-script/install_docker_composer.sh:9]: Newest release is 1.28.5

**Review by WikiRik — COMMENTED** [review]
New changes look good, just have some comments about versions that we are using in packages.

**Review by WikiRik — APPROVED** [review]
We'll look at the secrets and new releases of Ubuntu and other stuff later

**Pull request merged**